game/scripts/npc/items/item_bottle.txt

```diff
@@ -0,0 +1,67 @@
+    // Bottle
+    //=================================================================================================================
+
+    "item_bottle"
+    {
+        // General
+        //-------------------------------------------------------------------------------------------------------------
+        "ID"                            "41"                                                        // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
+        "AbilityBehavior"               "DOTA_ABILITY_BEHAVIOR_IMMEDIATE | DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_OPTIONAL_UNIT_TARGET"
+        "AbilityUnitTargetTeam"         "DOTA_UNIT_TARGET_TEAM_FRIENDLY"
+        "AbilityUnitTargetType"         "DOTA_UNIT_TARGET_HERO"
+        "Model"                         "models/props_gameplay/bottle_blue.vmdl"
+        "ModelAlternate"                "models/props_gameplay/bottle_empty.vmdl"
+        "FightRecapLevel"               "1"
+        "SideShop"                      "1"
+        
+        // Stats
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilityCooldown"               "0.5"
+        "AbilityCastRange"              "350"
+        "AbilityCastPoint"              "0.0"
+        
+        // Item Info
+        //-------------------------------------------------------------------------------------------------------------
+        "ItemCost"                      "600"
+        "ItemPurchasable"               "0"
+        "ItemSellable"                  "0"
+        "ItemShopTags"                  "consumable"
+        "ItemQuality"                   "consumable"
+        "ItemAliases"                   "bottle"
+        "ItemStackable"                 "0"
+        "ItemShareability"              "ITEM_PARTIALLY_SHAREABLE"
+        "ItemPermanent"                 "1"
+        "ItemInitialCharges"            "3"
+        "ItemDisplayCharges"            "1"
+        "IsTempestDoubleClonable"       "0"
+        
+        // Special  
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilitySpecial"
+        {
+            "01"
+            {
+                "var_type"              "FIELD_INTEGER"
+                "health_restore"        "90"
+            }
+            "02"
+            {
+                "var_type"              "FIELD_INTEGER"
+                "mana_restore"          "60"
+            }           
+            "03"
+            {
+                "var_type"              "FIELD_FLOAT"
+                "restore_time"          "2.5"
+            }
+            "04"
+            {
+                "var_type"              "FIELD_INTEGER"
+                "movement_speed_percent_bonus"          "-30"
```

Comment:
Why is this here? Does the bottle really reduce ms, or is it just an orphan stat?